src/main/java/seedu/letsgethired/model/InternTracker.java

```diff
@@ -110,6 +112,14 @@ public ObservableList<InternApplication> getApplicationList() {
         return internApplications.asUnmodifiableObservableList();
     }
 
+    public void setCurrentApplication(InternApplication currentApplication) {
```

Comment:
Same here and other instances of `currentApplication` - perhaps using `selectedInternApplication` would be more consistent with the rest of the code and less ambigous!